php/php.phpunit/src/org/netbeans/modules/php/phpunit/commands/PhpUnit.java

```diff
@@ -406,7 +407,12 @@ private TestParams getTestParams(PhpModule phpModule, TestRunInfo runInfo) throw
                 // standard suite
                 // #218607 - hotfix
                 //params.add(SUITE_NAME)
-                params.add(getNbSuite().getAbsolutePath());
+                if (PhpUnitPreferences.getPhpUnitVersion(phpModule) == PhpUnitVersion.PHP_UNIT_9) {
```

Comment:
Maybe we could add a method into `PhpUnitVersion`, e.g. `usesNbSuite()` and call it here?

Reply:
Yes, it's good way!